rules/S6169/cfamily/metadata.json

```
@@ -1,5 +1,5 @@
 {
-  "title": "Call to \"std::is_constant_evaluated\" should not be gratuitous",
+  "title": "Uses of \"std::is_constant_evaluated\" and \"if consteval\" should not be redundant",
```

When I read it, I understand that `std::is_constant_evaluated` should not be redundant with `if consteval`.
```suggestion
  "title": "\"std::is_constant_evaluated\" and \"if consteval\" should only be used when necessary",
```